crates/turbopack-core/src/resolve/options.rs

```diff
@@ -442,6 +442,9 @@ pub struct ResolveOptions {
     pub resolved_map: Option<Vc<ResolvedMap>>,
     pub before_resolve_plugins: Vec<Vc<Box<dyn BeforeResolvePlugin>>>,
     pub plugins: Vec<Vc<Box<dyn AfterResolvePlugin>>>,
+    /// Support resolving *.js requests to *.ts files
+    pub enable_js_ts_rewriting: bool,
```

Comment:
Maybe have a better name for that. Something with typescript.

`enable_typescript_with_output_extension`